crash-reporter/init.sls

```diff
@@ -0,0 +1,55 @@
+{% from 'common/map.jinja' import common %}
+{% from tpldir ~ '/map.jinja' import tracker %}
+
+include:
+  - common
+  - python
+
+crash-reporter:
+  virtualenv.managed:
+    - name: /home/servo/crash-reporter/_venv
+    - venv_bin: virtualenv-3.5
+    - python: python3
+    - system_site_packages: False
+    - require:
+      - pkg: python3
+      - pip: virtualenv
+  pip.installed:
+    - pkgs:
+      - git+https://github.com/servo/crash-reporter@{{ tracker.rev }}
+    - bin_env: /home/servo/crash-reporter/_venv
+    - upgrade: True
+    - require:
+      - virtualenv: crash-reporter
+  {% if grains.get('virtual_subtype', '') != 'Docker' %}
+  service.running:
+    - enable: True
+    - name: tracker
+    - require:
+      - pip: crash-reporter
+    - watch:
+      - file: /home/servo/crash-reporter/config.json
+      - file: /lib/systemd/system/tracker.service
+      - pip: crash-reporter
+  {% endif %}
+
+/home/servo/crash-reporter/config.json:
+  file.managed:
+    - source: salt://{{ tpldir }}/files/config.json
```

Comment:
For the shared secret, we will want to add:
```
- context:
    secret: {{ pillars['crash-reporter']['secret'] }}
```
We will also need to add /srv/pillar/crash-reporter.sls on the main machine, and add the following fake crash-reporter.sls to .travis/test_pillars/:
```
'crash-reporter':
  'secret': 'TEST-CRASH-REPORTER-SECRET'
```